library/retrieve_conjur_variable.py

```diff
@@ -160,16 +161,20 @@ def execute(self):
                 else:
                     self.exit_with_error('Conjur identity should be in environment variables or in one of the following paths: \'~/.netrc\', \'/etc/conjur.identity\'')
 
-            # Load our certificate for validation
-            ssl_context = ssl.create_default_context()
-            ssl_context.load_verify_locations(conf['cert_file'])
-            conjur_https = HTTPSConnection(urlparse(conf['appliance_url']).netloc,
-                                           context = ssl_context)
+            if conf['appliance_url'].startswith('https') is True:
+                    # Load our certificate for validation
+                    ssl_context = ssl.create_default_context()
+                    ssl_context.load_verify_locations(conf['cert_file'])
+                    conjur_connection = HTTPSConnection(urlparse(conf['appliance_url']).netloc,
+                                               context = ssl_context)
+            else:
+                    conjur_connection = HTTPConnection(urlparse(conf['appliance_url']).netloc)
```

Comment:
What happens if I don't enter a protocol? Let's say I write just "conjur:443"? SHould we accept this?

We may also check that it's "http/https" and if it's not any of these to throw a relevant exception